ultimate-dashboard.php

```diff
@@ -32,58 +36,59 @@ function udb_admin_scripts() {
 
 	global $pagenow, $typenow;
 
-	// Widget edit screen & create a new Widget screen
-	if( ( $pagenow == 'post.php' || $pagenow == 'post-new.php' ) && $typenow == 'udb_widgets' ) {
+	$plugin_data = get_plugin_data( __FILE__ );
```

Comment:
For some weird reasons, get_plugin_data sometimes is undefined. Do a function exist check like so https://wordpress.stackexchange.com/a/288773/59917

Reply:
Hi @collizo4sky , thanks for reviewing and for the suggestion!
Will do it, and might be part of next PR

Reply:
Hi @collizo4sky , i change my mind 😄 

In the link above he said "_If you need to use the get_plugin_data on the frontend..._"

And also the thread starter's problem situation was: he called `get_plugin_data` from a `__construct` inside a class which is included in `functions.php`.

I think that wouldn't be our case since im calling that function in a `admin_enqueue_scripts` hook inside a plugin in administration (settings) area. So, looks like checking is not necessary in this case.

Reply:
Checking is just a small if condition. Wouldn't hurt to add it 😄